trader/trader.go

```diff
@@ -534,3 +614,31 @@ func (t *Trader) getExistingOffers() ([]hProtocol.Offer /*sellingAOffers*/, []hP
 func (t *Trader) setExistingOffers(sellingAOffers []hProtocol.Offer, buyingAOffers []hProtocol.Offer) {
 	t.sellingAOffers, t.buyingAOffers = sellingAOffers, buyingAOffers
 }
+
+func (t *Trader) countOfferChangeTypes(offers []txnbuild.Operation) (numCreate int, numUpdate int, numDelete int, e error) {
```

Comment:
can you add a test for this?

- we can pass in 3 create offers, 2 updates, 1 delete and it should recognize it.
- then we should mix the order like (1 delete, 1 updates, 1 delete, 1 create, 2 update)
- a third case with just 2 update orders
- and a fourth case with just 1 create order

Reply:
note that once we move the function call to the correct place, the input will be of type `[]build.TransactionMutator` and not `[]txnbuild.Operation`.